examples/pixelcnn/train.py

```diff
@@ -210,32 +186,46 @@ def train(pcnn_module, model_dir, batch_size, init_batch_size, num_epochs,
   eval_iter = iter(eval_ds)
 
   # Compute steps per epoch and nb of eval steps
-  steps_per_epoch = data_source.TRAIN_IMAGES // batch_size
-  steps_per_eval = data_source.EVAL_IMAGES // batch_size
+  if config.num_train_steps == -1:
+    steps_per_epoch = (
+      data_source.info.splits['train'].num_examples // config.batch_size
+    )
+  else:
+    steps_per_epoch = config.num_train_steps
+
+  if config.num_eval_steps:
```

Comment:
should it be `if config.num_eval_steps == -1:` ?

Reply:
Done, thanks!